pyproject.toml

```diff
@@ -99,10 +99,11 @@ ignore="H017"
 "network-api/networkapi/templates/pages/publication_page.html" = "T003,T032"
 "network-api/networkapi/templates/pages/styleguide.html" = "H006,H013,H014,H026,T003,T032"
 "network-api/networkapi/templates/partials/404.html" = "D018,H006"
-"network-api/networkapi/templates/partials/banner.html" = "H006,H025,T003"
+"network-api/networkapi/templates/partials/donate_banner.html" = "H006,H025,T003"
```

Comment:
I think we should appreciate the opportunity to fix these instead of just replacing the [rules]( https://www.djlint.com/docs/linter/):
- `H006` is an image missing `height` and `width` attributes
- `H025` is an orphan tag (this one is a bit worrying)
- `T003` is that `endblock`s should have a name (just add the same name as the block declaration)

Reply:
I was following the example from the previous template. But you are right! We should 'Lead The Change', and 'Raise The Bar'. I have fixed these now.

Reply:
I'm guilty of doing the same before, don't worry :see_no_evil: thanks for fixing those